### PR TITLE
add: `google-earth-pro-stable-deb`

### PR DESCRIPTION
**Pull request by Zahrun**

### CODE REVIEW

File: packages/google-earth-pro-stable-deb/google-earth-pro-stable-deb.pacscript [hunk 1]
```diff
@@ -0,0 +1,7 @@
+name="google-earth-pro-stable-deb"
+pkgname="google-earth-pro-stable"
+version="7.3.4.8248"
+url="https://dl.google.com/dl/earth/client/current/google-earth-pro-stable_current_amd64.deb"
+breaks="${pkgname} ${pkgname}-git ${pkgname}-bin ${pkgname}-app ${pkgname}-deb "
```
Comment (Henryws) [inline packages/google-earth-pro-stable-deb/google-earth-pro-stable-deb.pacscript:5]: Remove extra space at the end

**Review by wizard-28 — CHANGES_REQUESTED** [review]
Add `maintainer` field

**Comment by Zahrun** [comment]
> Add `maintainer` field

I think one of the beauties of this project is that anyone can update the script when there is a new version.
I’m not sure that I’ll keep updated for new versions of this particular one.
So I thought it would be fine to have a warning message for people using it, saying it is not necessarily up to date.
Actually in this one, the hash will become invalid with every version, so it does not really apply.
Is it that when we add a package, we commit to keep it updated at all times?
I’m ok to put my name, but still wanted to have this discussion with you.

**Comment by wizard-28** [comment]
> Actually in this one, the hash will become invalid with every version, so it does not really apply.
Is it that when we add a package, we commit to keep it updated at all times?

Yes, that's how the whole updating system works, you'll have to make a PR for every version, when this package releases a new version.

> I’m ok to put my name, but still wanted to have this conversation with you.

If you do not wish to maintain your pacscript, then you can remove the `maintainer` field.

File: packages/ferdi-deb/ferdi-deb.pacscript [hunk 2]
```diff
@@ -2,7 +2,7 @@ name="ferdi-deb"
 pkgname="ferdi"
 version="5.6.3"
 url="https://github.com/getferdi/ferdi/releases/download/v${version}/ferdi_${version}_amd64.deb"
-breaks="${pkgname} ${pkgname}-git ${pkgname}-bin ${pkgname}-app ${pkgname}-deb "
+breaks="${pkgname} ${pkgname}-git ${pkgname}-bin ${pkgname}-app ${pkgname}-deb"
```
Comment (wizard-28) [inline packages/ferdi-deb/ferdi-deb.pacscript:5]: Why was this modified here. Make reverse this change, and make a separate PR for this.

**Review by wizard-28 — CHANGES_REQUESTED** [review]
Also rename your PR title to 
```
add: `google-earth-pro-stable-deb`
```

Also try to follow the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) system for commit messages.

The types available to use are: default types, and these two:
1. `add`: When you add a new package
2. `upd`: When you update a package

As you can see, the PR title should also follow the conventional commit message scheme

File: packages/ferdi-deb/ferdi-deb.pacscript (hunk 2 above)
Comment (Zahrun) [reply packages/ferdi-deb/ferdi-deb.pacscript:5]: The commit is "remove extra space at the end" as the line is exactly the same in the two packages, it was combined in one commit.

**Comment by Zahrun** [comment]
> > Actually in this one, the hash will become invalid with every version, so it does not really apply.
> > Is it that when we add a package, we commit to keep it updated at all times?
> 
> Yes, that's how the whole updating system works, you'll have to make a PR for every version, when this package releases a new version.

I meant especially this one, the url does not include the version number

File: packages/ferdi-deb/ferdi-deb.pacscript (hunk 2 above)
Comment (Zahrun) [reply packages/ferdi-deb/ferdi-deb.pacscript:5]: Thank you for the conventionnal commits system link, I didn’t know about it

Comment (Zahrun) [reply packages/ferdi-deb/ferdi-deb.pacscript:5]: You really want me to revert the commit, and create a separate commit for a space typo?

Comment (wizard-28) [reply packages/ferdi-deb/ferdi-deb.pacscript:5]: Let's try to keep the PR on topic (the topic is about adding `google-earth-pro-stable-deb`). Off-topic changes (though maybe discussed or brought up in this PR), which aren't hard-depended on by this PR should be made in their separate PR. This helps to maintain a clean commit history, as many (or rather one) of the users of pacstall uses a RSS feed to filter out changes.

So it's best to keep the changes on topic and code systematically.

Comment (Zahrun) [reply packages/ferdi-deb/ferdi-deb.pacscript:5]: Sorry I had not seen that the README.MD had changed and now includes stuff about pro-hooks and stuff.

Ok I will revert

**Comment by wizard-28** [comment]
We should implement:
```bash
hash=('SKIP')
```

For these pacscripts.

**Review by wizard-28 — CHANGES_REQUESTED** [review]
Add `gives`

File: packages/google-earth-pro-stable-deb/google-earth-pro-stable-deb.pacscript [hunk 3]
```diff
@@ -0,0 +1,9 @@
+name="google-earth-pro-stable-deb"
+pkgname="google-earth-pro-stable"
+version="7.3.4.8248"
+url="https://dl.google.com/dl/earth/client/current/google-earth-pro-stable_current_amd64.deb"
+breaks="${pkgname} ${pkgname}-git ${pkgname}-bin ${pkgname}-app ${pkgname}-deb"
+gives="${pkgname}"
+description="Renders a 3D representation of Earth based primarily on satellite imagery"
+hash="cdf406c0574564b2b1c2bdcfd731c45ae6358d1f69969895ccd4ea6344bc6add"
+maintainer="Zahrun <Zahrun@github.com>"
```
Comment (wizard-28) [inline packages/google-earth-pro-stable-deb/google-earth-pro-stable-deb.pacscript:9]: Didn't you want not to maintain this?

Comment (Zahrun) [reply packages/google-earth-pro-stable-deb/google-earth-pro-stable-deb.pacscript]: Sure

**Review by wizard-28 — APPROVED** [review]
